openvdb/openvdb/unittest/TestPointDataLeaf.cc

```diff
@@ -1186,7 +1186,7 @@ TEST_F(TestPointDataLeaf, testIO)
                 AttributeF::cast(leafFromDisk->constAttributeArray("density")));
 
             EXPECT_TRUE(leafFromDisk->buffer().isOutOfCore());
-#if OPENVDB_USE_BLOSC
+#ifdef OPENVDB_USE_BLOSC
```

Comment:
This was interesting. Doing `add_definition(-DOPENVDB_USE_BLOSC)` does not produce the same behaviour as `#define OPENVDB_USE_BLOSC`. With the prior you can do `#if OPENVDB_USE_BLOSC`, but this fails with the latter. I'm not sure if this is a problem at all.